src/Compatibility/Core/src/Android/AppCompat/Platform.cs

```diff
@@ -267,6 +268,9 @@ internal static SizeRequest GetNativeSize(VisualElement view, double widthConstr
 			}
 			else if ((visualElementRenderer == null || visualElementRenderer is HandlerToRendererShim) && view is IView iView)
 			{
+				Application.Current?.FindMauiContext()?.CreateLogger<Platform>()?.LogWarning(
+					"Someone called Platform.GetNativeSize instead of going through the Handler.");
+
 				returnValue = iView.Handler.GetDesiredSize(widthConstraint, heightConstraint);
 			}
```

Comment:
Keeping this chunk of code here for "just in case", but this code path should never be hit because this service is obsolete and could co away at any time. All callers need to check the handler before using ths service so a removal of renderers/compat is a seamless operation.